Rubberduck.VBEEditor/VBERuntime/Vb6NativeApi.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Rubberduck.VBEditor.VbeRuntime;
+
+namespace Rubberduck.VBEditor.VBERuntime
+{
+    internal class Vb6NativeApi : IVbeNativeApi
+    {
+        private const string _dllName = "vba6.dll";
```

Comment:
I still worry that this might not work if the vb6 loads `msvmvb6.dll` as it did in my instance. I could not locate the vbe6.dll in the PE unless I'm doing it wrong.

Reply:
Did it *only* load `msvmvb6.dll`? I could have sworn that pretty much everything that targets that run-time forwards calls through to it.

Reply:
when I looked, yes. I only created a empty vb6 exe project, then immediately created a test module. Ran PE and it didn't even show vbe6.dll unless I completely missed it.

Reply:
`vba6.dll` for `Vb6NativeApi`? I'm missing something.. or is it a typo?

Reply:
I also see that's something of a red herring --- VBA will load `vbe6` whereas VB6 will load `vba6`. Had to squint a bit more...

Reply:
> VBA will load vbe6 whereas VB6 will load vba6

An explanatory comment (with credits to MS?) to that effect might help relieving stuck eyebrows.